packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -49,7 +51,9 @@ export const DefaultCell = (props: TableCellProps) => {
 
   const isStringValue = typeof value === 'string';
 
-  const cellStyle = getCellStyle(tableStyles, cellOptions, displayValue, inspectEnabled, isStringValue);
+  // Text should wrap when the content length is less than or equal to the length of an OG tweet and it contains whitespace
+  const textShouldWrap = displayValue.text.length <= OG_TWEET_LENGTH && displayValue.text.search(/\s/) >= 0;
```

Comment:
`displayValue.text.search(/\s/) >= 0` can also be `/\s/.test(displayValue.text)`

Reply:
Ah nice, didn't know about `test`. Updated! 😀